test/language_data/PKG-INFO

```diff
@@ -0,0 +1,23 @@
+Metadata-Version: 2.1
```

Comment:
If I recall correctly this isn't the official package from Facebook, so it gives us a false positive with `ProductInfo` `(facebook, zstandard, 0.18.0)`

I think it doesn't really affects the test but just informing to make sure.

Reply:
Yes, it doesn't affect the test!
I just used the metadata of the zstd package we are using in the project